database/postgres/parser.go

```diff
@@ -1071,40 +1071,48 @@ func (p PostgresParser) parseTypeName(node *pgquery.TypeName) (parser.ColumnType
 		}
 	}
 
-	if len(typeNames) == 1 {
```

Comment:
This diff is a bit easier to read if you add `?w=1` to the end of the GitHub URL (or use `git diff -w` locally).